src/test/java/com/wikia/webdriver/testcases/specialpagestests/EditingPreferencesTests.java

```diff
@@ -1,11 +1,14 @@
 package com.wikia.webdriver.testcases.specialpagestests;
 
+import org.apache.commons.lang3.StringUtils;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'org.apache.commons.lang3.StringUtils'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)